frontend/lib/src/components/widgets/Selectbox/Selectbox.test.tsx

```diff
@@ -130,8 +128,10 @@ describe("Selectbox widget", () => {
     props.widgetMgr.submitForm("form", undefined)
 
     // Our widget should be reset, and the widgetMgr should be updated
-    expect(screen.getByText("a")).toBeInTheDocument()
-    expect(screen.queryByText("b")).not.toBeInTheDocument()
+    expect(await screen.findByText("a")).toBeInTheDocument()
+    await waitFor(() =>
```

Comment:
was this only passing via winning a race condition? (before this change, which i agree with and feel like we've been spoiled with e2e testing frameworks doing this for us)

Reply:
I think it had to do with the old `setState` behavior. But actually, your comment made me realize that this test should be be utilizing [React act](https://react.dev/reference/react/act) instead of updating these RTL query call types. Will push a revision with that change instead.